experimental/terraform/modules/fourkeys-bigquery/queries/changes.sql

```diff
@@ -0,0 +1,10 @@
+# Changes Table
```

Comment:
In general, I would prefer to keep the `queries` as a top level directory so they're easy to find.  Is that possible with TF modularization?